Addressed by updating the PR description (no fix-up commit needed — description-only):

- "Summary" line now says the directory bind mount landed in `main` as #77 (was: "depends on #74").
- Test-plan `make test-e2e` line now says "gated on #79" with a one-line description of the failure mode, instead of "gated on #74".

`make test-e2e` evidence on the rebased branch (head `0d27a027`, freshly rebuilt `kuke`):

```
=== RUN   TestKuke_AttachDetach_KeepsTaskRunning
    e2e_kuke_attach_test.go:154: sbsh socket "<runPath>/<realm>/<space>/<stack>/<cell>/work/tty/socket" did not appear within 10s
--- FAIL: TestKuke_AttachDetach_KeepsTaskRunning (23.18s)
```

`apply` exits 0 but `ctr -n <realm> task ls` is empty for the work container, and starting it manually surfaces the underlying error from #79:

```
$ sudo ctr -n dbg-realm task start dbg-space_dbg-stack_dbg-cell_work
Error: unknown flag: --terminal-socket
```

OCI spec for the work container *does* now match #69's source (directory mount of `tty/`, `--run-path`/`--terminal-socket`/`--capture-file`/`--terminal-logfile` wrapper args), so the #77 prerequisite is genuinely satisfied — the residual failure is purely the wrapper flag set vs. the host `sbsh terminal` subcommand:

```
$ sbsh terminal --run-path /tmp --terminal-socket /tmp/x --capture-file /tmp/y --terminal-logfile /tmp/z -- sleep 1
Error: unknown flag: --terminal-socket
```

`sbsh terminal --help` exposes `--socket`, `--log-file`, `--command`. That is the exact bug #79 documents.

Other e2e tests in `make test-e2e` failed on this run too, but with `dial kukeond at /run/kukeon/kukeond.sock: ... no such file or directory` — the host daemon socket is gone in this environment, unrelated to this PR. The new attach test runs `--no-daemon` so it isn't affected by that.

Switching back to `ci-stalled` since #79 is the actual gate and there's no code change in this PR that can move it.

Deferred non-blocking nits the reviewer flagged previously (will roll into a follow-up rebase once #79's wrapper fix lands and `make test-e2e` can go green here):
- `verifyContainerTaskIsRunning` `t.Skipf` on missing `ctr` instead of returning false.
- `attachConnectGrace = 2s` fixed-sleep — leave as-is, monitor for flakes.

--- a/e2e/e2e_kuke_attach_test.go
+++ b/e2e/e2e_kuke_attach_test.go
@@ -65,6 +65,7 @@ func stageHostSbsh(t *testing.T, runPath string) {
 	if err != nil {
 		t.Skipf("sbsh binary not found on PATH: %v", err)
 	}
+	requireSbshHasPostMergeFlags(t, hostSbsh)
 
 	dstDir := filepath.Join(fullRunPath, ctrpkg.SbshCacheSubdir, runtime.GOARCH)
 	if err = os.MkdirAll(dstDir, 0o755); err != nil {
@@ -86,6 +87,24 @@ func stageHostSbsh(t *testing.T, runPath string) {
 
 	if _, err = io.Copy(out, src); err != nil {
 		t.Fatalf("copy sbsh %q -> %q: %v", hostSbsh, dst, err)
+	}
+}
+
+// requireSbshHasPostMergeFlags ensures the host sbsh carries the flag set
+// the wrapper injected at #69 assumes (post sbsh#153, which added
+// `--capture-file` to the `sbsh terminal` subcommand). Without this guard a
+// stale sbsh on PATH manifests as `unknown flag: ...` deep inside the work
+// container's task, surfaced here only as a `waitForSocket` timeout — hard
+// to attribute. Skipping with a named cause saves the dig.
+func requireSbshHasPostMergeFlags(t *testing.T, hostSbsh string) {
+	t.Helper()
+	out, err := exec.Command(hostSbsh, "terminal", "--help").CombinedOutput()
+	if err != nil {
+		t.Fatalf("sbsh terminal --help failed: %v\noutput:\n%s", err, out)
+	}
+	if !strings.Contains(string(out), "--capture-file") {
+		t.Skipf("host sbsh %q lacks `sbsh terminal --capture-file` (sbsh#153); "+
+			"upgrade sbsh on PATH and retry", hostSbsh)
 	}
 }
 

